rotary/rules/circularize.smk

```diff
@@ -33,18 +33,15 @@ rule download_hmm:
 # Based on clustering tutorial at https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html (accessed 2022.3.31)
 checkpoint split_circular_and_linear_contigs:
     input:
-        assembly_stats="{sample}/assembly/{sample}_circular_info.tsv",
-        filter_list="{sample}/polish/cov_filter/{sample}_filtered_contigs.list"
+        "{sample}/polish/{sample}_contig_info.tsv"
```

Comment:
This checkpoint now using the new contig summary report made during coverage filtration.

Reply:
Great